src/payment/cart/UpdateQuantityForm.jsx

```diff
@@ -4,8 +4,8 @@ import { connect } from 'react-redux';
 import { FormattedMessage } from '@edx/frontend-i18n';
 import { StatefulButton, Input } from '@edx/paragon';
 
-import { updateQuantity } from './data/actions';
-import { updateQuantityFormSelector } from './data/selectors';
+import { updateQuantity } from '../data/actions';
+import { updateQuantityFormSelector } from '../data/selectors';
```

Comment:
This is a nit, but is maybe interesting to think about... technically this breaks the 'rules' of importing something from your parent - i.e., taking for granted something outside your module/context.  If I were to try to avoid that, I might pass the action in as a parameter to this component.  The selector is harder, as it depends on selectors in the parent too.  I think we'd have to jump through some hoops to get the selector to be context agnostic, and it's clearly not worth it here.  But, that said, if the selector were in this sub-directory's data dir instead of in the parent, that'd at least be marginally closer.  ¯\_(ツ)_/¯ 

This all has me thinking that there should probably be a difference between a module and a sub-directory merely for organization.  It also reminds me that redux breaks all rules of encapsulation and modularity by forcing components to be aware of the global position of their data. 😡

Reply:
Hah yea, I do think it would be good to make an obvious distinct when a folder house s a module vs just organizes some code.